package/yast2.changes

```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu May 14 07:32:16 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- Re-implement XML parser (bsc#1170886):
+-- use as backend nokogiri instead of hand written libxml2 wrapper
```

Comment:
So are we able to drop the old one?

Reply:
not right now, as there are one module in autoYaST that calls directly xml agent, but plan is to replace it with XML module.

Reply:
OK, it is https://github.com/yast/yast-autoinstallation/blob/f61e00734ee8c6caa7810a0ffd48afb422a31281/src/modules/AutoinstClass.rb#L55

Then there is https://github.com/yast/yast-update/blob/77b9d0f206002e3e34eccab8beace13f1c5b59e2/src/modules/RootPart.rb#L1214 using `.anyxml`, https://github.com/yast/yast-yast2/blob/master/library/general/src/servers_non_y2/ag_anyxml

Reply:
good catches. I think we should simply drop that ag_anyxml and use nokogiri directly as it is used for pam, so not "our" xml